knotx-it-tests/src/test/resources/content/local/customAndDefaultSymbol.html

```diff
@@ -56,9 +56,7 @@ <h2>Snippet1 - {[_result.message]}</h2>
       <div class="col-md-4">
         <h2>Snippet1 - {[second._result.message]}</h2>
```

Comment:
Can we make the custom simple more visible like `&&value&&` instead of `{[value]}`.

Reply:
Actually it makes more sense to move tests with custom delimiter to `knotx-template-engine` module.